.github/workflows/stale.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - uses: actions/stale@v9
         with:
           debug-only: ${{ inputs.dryRun || false }}
-          operations-per-run: ${{ inputs.operationsPerRun || 100 }}
+          operations-per-run: ${{ inputs.operationsPerRun || 500 }}
```

Comment:
`operationsPerRun` has default value of `30`, so does `500` works here?

https://github.com/apache/kafka/blob/trunk/.github/workflows/stale.yml#L30

Reply:
I assumed it is overridden because of the message @mumrah shared:
> Github API rate used: 101